ui/panels-plugin/src/plugins/stat-chart/stat-chart-model.ts

```diff
@@ -42,5 +44,6 @@ export function createInitialStatChartOptions(): StatChartOptions {
       kind: 'Decimal',
     },
     sparkline: {},
+    fontSize: 'Default',
```

Comment:
i think it's ok to remove this since fontSize is optional

Reply:
Removed!